Cargo.toml

```diff
@@ -4,4 +4,11 @@ version = "0.1.0"
 edition = "2021"
 
 [build-dependencies]
+bindgen = "0.71.1"
```

Comment:
I think we should do something like this here (unsure if the syntax is fully correct)
```
[target.'cfg(feature = "build-clay")'.build-dependencies]
bindgen = "0.71.1"
```
So bindgen is only included (and compiled) when the feature toggle is enabled.